tensorflow/tools/docs/generate_lib.py

```diff
@@ -133,7 +133,7 @@ def write_docs(output_dir,
 
```

Comment:
Sorry for the delay. It looks like the docs tool is being moved to a separate package, and the owners prefer not to make changes until that's complete. Could you remove this file only?

Reply:
Removed!

Reply:
Thanks! It still shows in the PR, as deleted. We need to exclude any changes to it from the PR.

Reply:
How do you mean, exactly? Squash the two commits to one?

Reply:
I believe `git reset` should work. `git revert` + squash might work too.

Reply:
How does it look now?

Reply:
Still shows deleted, but maybe the UI is confusing me. Let's try to pull it and see if the tests are happy.

Reply:
I'm a bit confused, when you said to remove `tensorflow/tools/docs/generate_lib.py`, I understood you meant to delete it.

Did you instead mean to not make any changes to that file in this PR?

Thanks!

Reply:
Yes, sorry for the confusion. I guess I should have said to revert it.

Reply:
No problem at all! How's it look now?

Reply:
Looks good now, thank you!